terraform/20-app/cognito.tf

```diff
@@ -0,0 +1,120 @@
+resource "aws_iam_role" "cognito_sns_role" {
+  name = "${local.prefix}-cognito-sns-role"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [
+      {
+        Effect = "Allow",
+        Principal = {
+          Service = "cognito-idp.amazonaws.com"
+        },
+        Action = "sts:AssumeRole"
+      }
+    ]
+  })
+}
+
+resource "aws_sns_topic" "cognito_topic" {
```

Comment:
What was your thinking behind building the corresponding SNS outside of the cognito module?

Reply:
removed this